Jenkinsfile

```diff
@@ -0,0 +1,59 @@
+pipeline {
+  agent {
+    kubernetes {
+      yamlFile 'kubes-pod.yaml'
+      defaultContainer 'ubuntu'
+      activeDeadlineSeconds 3600
+      idleMinutes 15
+    }
+  }
+  stages {
+    stage('Init') {
+      steps {
+        checkout scm
+	script {
+	  def m = (env.GIT_URL =~ /(\/|:)(([^\/]+)\/)?(([^\/]+?)(\.git)?)$/)
+	  if (m) {
+	    org = m.group(3)
+	    repo = m.group(5)
+	  }
+	}
+      }
+    }
+    stage('Setup build') {
+      when { not { buildingTag() } }
+      steps {
+        script {
+          target = "verify"
+        }
+      }
+    }
+    stage('Setup release') {
+      when { buildingTag(); not { changeRequest() }  }
+      steps {
+        script {
+          target = "deploy -P release -P rpm --settings settings.xml"
```

Comment:
Does the non-existent rpm profile matter? What about including the logging options from below?

Reply:
I don't think the rpm stuff mattered, but I removed it anyway. The logging options should still be good.